diff --git a/dunstrc b/dunstrc
index c70de40..bc85e2b 100644
--- a/dunstrc
+++ b/dunstrc
@@ -24,16 +24,17 @@
     # dynamic width from 0 to 300
     # width = (0, 300)
     # constant width of 300
-    width = 300
+    # Use huuuuge number to auto use maximum available with
+    width = 1000000
 
     # The maximum height of a single notification, excluding the frame.
     height = 300
 
     # Position the notification in the top right corner
-    origin = top-right
+    origin = top-left
 
     # Offset from the origin
-    offset = 10x50
+    offset = 0x0
 
     # Scale factor. It is auto-detected if value is 0.
     scale = 0
@@ -86,10 +87,10 @@
 
     # Defines width in pixels of frame around the notification window.
     # Set to 0 to disable.
-    frame_width = 3
+    frame_width = 1
 
     # Defines color of the frame around the notification window.
-    frame_color = "#aaaaaa"
+    frame_color = "#72B3CC"
 
     # Define a color for the separator.
     # possible values are:
@@ -107,11 +108,11 @@
     # Set to 0 to disable.
     # A client can set the 'transient' hint to bypass this. See the rules
     # section for how to disable this if necessary
-    # idle_threshold = 120
+    idle_threshold = 120
 
     ### Text ###
 
-    font = Monospace 8
+    font = Monospace 14
 
     # The spacing between lines.  If the height is smaller than the
     # font height, it will get raised to the font height.
@@ -151,7 +152,7 @@
     #   %n  progress value if set without any extra characters
     #   %%  Literal %
     # Markup is allowed
-    format = "<b>%s</b>\n%b"
+    format = "<b>%s</b>\n%b\n(%a)"
 
     # Alignment of message text.
     # Possible values are "left", "center" and "right".
@@ -196,7 +197,7 @@
     max_icon_size = 32
 
     # Paths to default icons.
-    icon_path = /usr/share/icons/gnome/16x16/status/:/usr/share/icons/gnome/16x16/devices/
+    icon_path = /usr/share/icons/gnome/32x32/emblems/:/usr/share/icons/gnome/32x32/status/:/usr/share/icons/gnome/32x32/actions/:/usr/share/icons/gnome/32x32/emotes/:/usr/share/icons/gnome/32x32/apps/:/usr/share/icons/gnome/32x32/devices/
 
     ### History ###
 
@@ -292,24 +293,23 @@
 [urgency_low]
     # IMPORTANT: colors have to be defined in quotation marks.
     # Otherwise the "#" and following would be interpreted as a comment.
-    background = "#222222"
-    foreground = "#888888"
+    background = "#000000"
+    foreground = "#72B3CC"
     timeout = 10
     # Icon for notifications with low urgency, uncomment to enable
     #default_icon = /path/to/icon
 
 [urgency_normal]
-    background = "#285577"
-    foreground = "#ffffff"
+    background = "#000000"
+    foreground = "#72B3CC"
     timeout = 10
     # Icon for notifications with normal urgency, uncomment to enable
     #default_icon = /path/to/icon
 
 [urgency_critical]
-    background = "#900000"
-    foreground = "#ffffff"
-    frame_color = "#ff0000"
-    timeout = 0
+    background = "#000000"
+    foreground = "#E09690"
+    timeout = 30
     # Icon for notifications with critical urgency, uncomment to enable
     #default_icon = /path/to/icon
 
@@ -433,4 +433,28 @@
 #    appname = "some_volume_notifiers"
 #    set_stack_tag = "volume"
 #
+
+[networkmanager]
+    appname = "NetworkManager"
+    urgency = low
+    new_icon = network-wireless
+    timeout = 2
+[james]
+    summary = "*James Greenwood*"
+    foreground = "#E09690"
+    urgency = critical
+    timeout = 30
+    format = "<b>%s</b>(%a)"
+[reuter-export]
+    summary = "TYPO3 Export"
+    foreground = "#8EB33B"
+    timeout = 30
+[switching]
+    summary = "switching*"
+    foreground = "#8EB33B"
+    timeout = 3
+[done]
+    summary = "*done*"
+    foreground = "#8EB33B"
+    timeout = 60
 # vim: ft=cfg
