diff --git a/autoload/test/php/phpunit.vim b/autoload/test/php/phpunit.vim
index 066aa3c..beee721 100644
--- a/autoload/test/php/phpunit.vim
+++ b/autoload/test/php/phpunit.vim
@@ -51,8 +51,6 @@ function! test#php#phpunit#executable() abort
     return g:test#php#phpunit#executable
   elseif filereadable('./vendor/bin/sail')
     return './vendor/bin/sail test'
-  elseif filereadable('./vendor/bin/paratest')
-    return './vendor/bin/paratest'
   elseif filereadable('./vendor/bin/phpunit')
     return './vendor/bin/phpunit'
   elseif filereadable('./bin/phpunit')
